WordPress/openverse
# Remove focused playwright test so all tests run; fix broken tests

## Fixes



Fixes #2191 by @obulat 🤞 

## Description


~~This PR reverts the changes to the Playwright config introduced in #1920 in an attempt to fix Playwright e2e tests.~~

~~I do not understand what in these changes makes Playwright not find all of the tests. @sarayourfriend, do you have an idea?~~

We missed the `.only` added in one of the tests, which caused Playwright to only run that one test...

## Testing Instructions

Check the CI Playwright e2e tests log - it should have around 155 tests (instead of 1).

Locally, you can use `just frontend/run test:playwright -- e2e` to run the tests.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
